bucket_A4/toxic/patches/patch-cfg_checks_python.mk

```diff
@@ -0,0 +1,22 @@
+--- cfg/checks/python.mk.orig	2018-02-27 23:46:56 UTC
++++ cfg/checks/python.mk
+@@ -1,15 +1,15 @@
+ # Variables for Python scripting support
+-PYTHON3_LIBS = python3
++PYTHON3_LIBS = python-3.7
+ PYTHON_CFLAGS = -DPYTHON
+ PYTHON_OBJ = api.o python_api.o
+ 
+ # Check if we can build Python scripting support
+ CHECK_PYTHON3_LIBS = $(shell $(PKG_CONFIG) --exists $(PYTHON3_LIBS) || echo -n "error")
+ ifneq ($(CHECK_PYTHON3_LIBS), error)
+-    LDFLAGS += $(shell python3-config --ldflags)
+-    CFLAGS += $(PYTHON_CFLAGS) $(shell python3-config --includes)
++    LDFLAGS += $(shell python3.7-config --ldflags)
```

Comment:
same thing, you can't hardcode this.  As soon as python default changes, this port breaks.